src/annotator/anchoring/range-js.js

```diff
@@ -0,0 +1,34 @@
+import { findChild, simpleXPathJQuery, simpleXPathPure } from './xpath-util';
```

Comment:
This is only called range-js so it does not collide with range.coffee imports. This will be renamed after range.coffee is pulled out.